integration_tests/fixtures/applicationData.json

```diff
@@ -1,5 +1,13 @@
 {
   "basic-information": {
+    "confirm-your-details": {
+      "officeLocation": "Sheffield",
```

Comment:
I think it might be worth removing `officeLocation` as Sarah thinks we don't need it, and we can replace it with Team / Region soon.